shared/src/validation-schemas/shownote/shownote.validation-schema.ts

```diff
@@ -0,0 +1,38 @@
+import { Joi } from '~/helpers/helpers';
+import {
+  ShownoteCreatePayloadKey,
+  ShownoteValidationMessage,
+  ShownoteValidationRule,
+} from '~/common/enums/enums';
+
+const shownote = Joi.object({
```

Comment:
we need to create separate validation
for backend & frontend
in shared folder we need to keep only common fields